@GordonDiggs yeah, agree. Would have been a better side to err on. Updated.

Explained in commit, but a parsing error will also have that information output. Example:

```
$ codeclimate analyze -e gofmt timecard/
Starting analysis
Running gofmt: Done!
error: (CC::Analyzer::Engine::EngineFailure) engine gofmt:stable failed with status 1 and stderr 
Analyzing path: /code/timecard/entry.go
Error analyzing path: /code/timecard/entry.go
Error: exit status 2
Gofmt output: /code/timecard/entry.go:9:1: illegal character U+0023 '#'
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,8 +1,14 @@
-FROM yunspace/alpine-golang
+FROM golang:alpine
 
-ADD build/codeclimate-gofmt /usr/src/app/
+MAINTAINER Code Climate <hello@codeclimate.com>
 
 RUN adduser -u 9000 -D app
 USER app
+
+WORKDIR /code
+
+COPY build/codeclimate-gofmt /usr/src/app/
+
+VOLUME /code
 
 CMD ["/usr/src/app/codeclimate-gofmt"]

--- a/bin/build
+++ b/bin/build
@@ -13,6 +13,6 @@ BUILD_SCRIPT="cd /src && "\
 "rm -rf build && mkdir build && "\
 "go build -o build/codeclimate-gofmt ."
 
-docker run -v "$SRC_ROOT:/src" yunspace/alpine-golang /bin/sh -c "$BUILD_SCRIPT"
+docker run -v "$SRC_ROOT:/src" golang:latest /bin/sh -c "$BUILD_SCRIPT"
 
 docker build -t "$IMAGE" .

--- a/codeclimate-gofmt.go
+++ b/codeclimate-gofmt.go
@@ -1,22 +1,27 @@
 package main
 
-import "github.com/codeclimate/cc-engine-go/engine"
-import "sourcegraph.com/sourcegraph/go-diff/diff"
-import "os/exec"
-import "strconv"
-import "os"
-import "strings"
+import (
+	"github.com/codeclimate/cc-engine-go/engine"
+	"sourcegraph.com/sourcegraph/go-diff/diff"
+	"os/exec"
+	"strconv"
+	"os"
+	"strings"
+	"fmt"
+)
 
 func main() {
 	rootPath := "/code/"
 
 	config, err := engine.LoadConfig()
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error loading config: %s", err)
 		os.Exit(1)
 	}
 
 	analysisFiles, err := engine.GoFileWalk(rootPath, engine.IncludePaths(rootPath, config))
 	if err != nil {
+		fmt.Fprintf(os.Stderr, "Error initializing: %s", err)
 		os.Exit(1)
 	}
 

--- a/codeclimate-gofmt.go
+++ b/codeclimate-gofmt.go
@@ -39,14 +39,16 @@ func main() {
 		}
 
 		if diffs != nil && diffs[0] != nil && len(diffs[0].Hunks) > 0 {
-			numHunks := strconv.Itoa((len(diffs[0].Hunks)))
+			numHunks := len(diffs[0].Hunks)
+
+			description := "Your code does not pass gofmt in " + strconv.Itoa(numHunks) + " " + pluralizePlace(numHunks) + ". Go fmt your code!"
 			path := strings.SplitAfter(path, rootPath)[1]
 
 			issue := &engine.Issue{
 				Type:              "issue",
 				Check:             "GoFmt/Style/GoFmt",
-				Description:       "Your code does not pass gofmt in " + numHunks + " places. Go fmt your code!",
-				RemediationPoints: int32(50000 * len(diffs[0].Hunks)),
+				Description:       description,
+				RemediationPoints: int32(50000 * numHunks),
 				Categories:        []string{"Style"},
 				Location: &engine.Location{
 					Path: path,
@@ -61,4 +63,12 @@ func main() {
 	}
 
 	os.Exit(0)
+}
+
+func pluralizePlace(quant int) string {
+	if quant > 1 {
+		return "places"
+	} else {
+		return "place"
+	}
 }

